clients/drcachesim/reader/v2p_reader.h

```diff
@@ -32,27 +32,41 @@
 
 /* v2p_reader: reads and parses a virtual-to-physical address mapping in textproto format.
  * Creates virtual-to-physical address map in memory.
+ * The section of the textproto file that we are interested in parsing is a sequence of
+ * blocks that follow this format:
+ * address_mapping {
+    virtual_address: 0x123
+    physical_address: 0x3
+ * }
+ * In gen_v2p_map() we rely on the fact that virtual_address and physical_address are one
+ * after the other on two different lines.
  */
 
 #ifndef _V2P_READER_H_
 #define _V2P_READER_H_ 1
 
-#include <stdint.h>
+#include "trace_entry.h"
 
 #include <cstdint>
-#include <unordered_map>
 #include <string>
+#include <unordered_map>
 
 namespace dynamorio {
 namespace drmemtrace {
 
+struct v2p_info_t {
+    uint64_t page_count;
```

Comment:
Add "=0" for default values for these 3 POD fields?

Reply:
Done.